<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1155</width>
    <height>723</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Info</string>
  </property>
  <widget class="QGroupBox" name="info_output_groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>761</width>
     <height>581</height>
    </rect>
   </property>
   <property name="title">
    <string>INFO_OUTPUT</string>
   </property>
   <widget class="QTableWidget" name="info_table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>731</width>
      <height>541</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <column>
     <property name="text">
      <string>이름</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>아이디</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>이메일</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QGroupBox" name="info_groupBox">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>50</y>
     <width>321</width>
     <height>581</height>
    </rect>
   </property>
   <property name="title">
    <string>INFO</string>
   </property>
   <widget class="QComboBox" name="info_comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>입력</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>검색</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>삭제</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="insert_pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>291</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>입력</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit_pushButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>540</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>종료</string>
    </property>
   </widget>
   <widget class="QLabel" name="id_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>45</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>아이디</string>
    </property>
   </widget>
   <widget class="QLabel" name="email_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>45</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>이메일</string>
    </property>
   </widget>
   <widget class="QLabel" name="name_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>40</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>이름</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_text">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="id_text">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="email_text">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>310</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exit_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1024</x>
     <y>601</y>
    </hint>
    <hint type="destinationlabel">
     <x>758</x>
     <y>667</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>insert_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>add_info()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>839</x>
     <y>469</y>
    </hint>
    <hint type="destinationlabel">
     <x>566</x>
     <y>643</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>add_info()</slot>
 </slots>
</ui>
